src/main/java/ui/UI.java

```diff
@@ -350,14 +351,18 @@ private Parent createRootNode() {
     public Set<String> getCurrentlyUsedRepos() {
         Set<String> currentlyUsedRepos = new HashSet<>();
         String defaultRepo = logic.getDefaultRepo();
-        currentlyUsedRepos.add(defaultRepo);
         currentlyUsedRepos.addAll(panels.getRepositoriesReferencedOnAllPanels());
+        if (!currentlyUsedRepos.stream().map(String::toLowerCase)
+                .collect(Collectors.toSet()).contains(defaultRepo.toLowerCase())) {
+            currentlyUsedRepos.add(defaultRepo);
+        }
```

Comment:
`getCurrentlyUsedRepos` could have a method's comment mentioning a post-condition that all repo ids are normalized (spaces removed/lowercase etc.). Maybe we should have some sorts of a separate `normalize` method in `Utility` that does all the repo clean up. But this all these won't be necessary anymore if we solve #1078